packages/trader/src/Modules/Contract/Containers/contract-replay.tsx

```diff
@@ -0,0 +1,215 @@
+            setIsVisible(false);
+            onUnmount();
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [contract_id, location, onMount, onUnmount]);
+
+    const onClickClose = React.useCallback(() => {
+        setIsVisible(false);
+        const is_from_table_row = !isEmptyObject(state) ? state.from_table_row : false;
+        return is_from_table_row ? history.goBack() : routeBackInApp(history);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [history, routeBackInApp]);
+
+    React.useEffect(() => {
+        // don't open Contract details page for trade types with disabled feature flag:
+        if (is_trade_type_disabled && is_visible) {
+            onClickClose();
+        }
+    }, [is_trade_type_disabled, is_visible, onClickClose]);
+
+    const onChangeSwipeableIndex = (index: number) => {
+        setSwipeIndex(index);
+    };
+
+    if (!contract_info.underlying) return null;
+
+    const is_accumulator = isAccumulatorContract(contract_info.contract_type);
+    const is_multiplier = isMultiplierContract(contract_info.contract_type);
+    const is_turbos = isTurbosContract(contract_info.contract_type);
+    const is_vanilla = isVanillaContract(contract_info.contract_type);
+    const is_smarttrader_contract = isSmartTraderContract(contract_info.contract_type);
+
+    const contract_drawer_el = (
+        <ContractDrawer
+            contract_info={contract_info}
+            contract_update={contract_update}
+            contract_update_history={contract_update_history}
+            is_accumulator={is_accumulator}
+            is_dark_theme={is_dark_theme}
+            is_market_closed={is_market_closed}
+            is_multiplier={is_multiplier}
+            is_turbos={is_turbos}
+            is_sell_requested={is_sell_requested}
+            is_vanilla={is_vanilla}
+            is_smarttrader_contract={is_smarttrader_contract}
+            onClickCancel={onClickCancel}
+            onClickSell={onClickSell}
+            status={indicative_status}
+            toggleHistoryTab={toggleHistoryTab}
+        />
+    );
+
+    const unsupportedContractOnConfirm = () => {
+        history.goBack();
+    };
+
+    const unsupportedContractOnClose = () => {
+        const statementws_url = urlFor('user/statementws', { legacy: true });
+        window.open(statementws_url, '_blank');
+    };
+
+    return (
+        <FadeWrapper is_visible={is_visible} className='contract-details-wrapper' keyname='contract-details-wrapper'>
+            <MobileWrapper>
+                <NotificationMessages />
+            </MobileWrapper>
+            <UnsupportedContractModal
+                onConfirm={unsupportedContractOnConfirm}
+                onClose={unsupportedContractOnClose}
+                is_visible={is_forward_starting}
+            />
+            <PageOverlay
+                id='dt_contract_replay_container'
+                header={localize('Contract details')}
```

Comment:
@kate-deriv can we use <Localize here? i can see that a node can be accepted in this prop :)